docs/contrib/tutes/setup-linux.rst

```diff
@@ -27,11 +28,17 @@ You're good to go.
 Opinionated Recommendations
 ===========================
 
-* Install Python with `pyenv`_
+* Install Python with pyenv
```

Comment:
reStructured text needs a new line here before sub dot points. ... It's write like that...